react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+export default function Accordion({ topics }) {
+  const [id, setId] = useState(0);
+
+  function handleID(buttonId) {
+    setId(buttonId);
+  }
+
+  return (
+    <>
+      <Header topics={topics} onClick={handleID} id={id} />
+    </>
+  );
+}
+
+function Header({ onClick, topics, id }) {
```

Comment:
I probably would not use this Header component at all and just put all this code inside Accordion. The reason is that this is where all your logic is, so the Accordion class isn't doing much. But, nonetheless, it works! Congratulations!